autogen/main/cluster.tf.tmpl

```diff
@@ -469,6 +469,13 @@ resource "google_container_cluster" "primary" {
       topic   = var.notification_config_topic
     }
   }
+  {% if beta_cluster %}
+  node_pool_auto_config {
```

Comment:
Could we make this a dynamic block enabled only if var.cluster_autoscaling.enabled for beta clusters?